addons/udes_warehouse_classification/tests/test_product_template.py

```diff
@@ -0,0 +1,59 @@
+from .common import Base
+
+
+class TestWarehouseClassificationPicking(Base):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.test_messages_01 = ["Inbound, Report a, Report"]
+        cls.test_messages_02 = ["Report a Report", "Report b Report"]
+        cls.test_messages_03 = ["Report a Report", "Inbound, Report a, Report"]
+
+    # functions to test:  and _get_classification_messages_for_product_picking
+
+    # For a given product find the classification messages for specified report,
+    # 1. report a on it's own - banana
```

Comment:
Either put this as the docstring of the class or add it to the docstring of each test :)